src/main/kotlin/org/rust/lang/refactoring/implementMembers/impl.kt

```diff
@@ -80,28 +79,26 @@ private fun insertNewTraitMembers(selected: Collection<RsAbstractable>, existing
         }
 
         val anchor = indexedExistingMembers
-            .map { IndexedValue(it.index, it.value.first) }
-            .lastOrNull()
+            .map {
+                val member = it.value.first
+                IndexedValue(it.index, member.expandedFromRecursively ?: member)
+            }.lastOrNull()
             ?: IndexedValue(-1, existingMembers.lbrace) ?: return
```

Comment:
This is strange `return`; do we really need it? Constructor cannot return nullable value (I guess)